schema/resolvers.js

```diff
@@ -1,21 +1,31 @@
-const { merge, includes } = require("lodash");
-const GraphQLDate = require("graphql-iso-date").GraphQLDate;
+const { GraphQLDate } = require("graphql-iso-date");
+const { includes } = require("lodash");
+
+const getId = args => args.newId || args.id;
+const getNewId = entity => entity.id.toString(10);
+const getInputArgs = args => args.input || args;
+const getIdFromArgs = args => getInputArgs(args).id;
```

Comment:
Would there ever be a need to check for `newId` here? e.g. `const getIdFromArgs = args => getId(getInputArgs(args));`

Reply:
No, should be fine. This is only used for mutations, there the named arg is called `id` and the field on the input type is also `id`